packages/lang-server/src/usage.ts

```diff
@@ -43,18 +43,17 @@ const getElemIDUsages = async (
   if (isInstanceElement(element) && element.refType.elemID.isEqual(id)) {
     pathesToAdd.add(element.elemID.getFullName())
   }
-  const transformFunc = ({ value, path }: TransformFuncArgs): Value => {
-    if (isReferenceExpression(value) && path) {
+  const func = ({ value, path }: WalkOnFuncArgs): Value => {
```

Comment:
this signature is not correct (`func` does not return `Value`)
```suggestion
  const func: WalkOnFunc = ({ value, path }) => {
```
I think we should not export the type `WalkOnFuncArgs` at all, I can't think of a reason to use it over `WalkOnFunc`